lib/google-wallet-hero-generator.js

```
@@ -0,0 +1,251 @@
+      // Load logo for watermark
+      const logoPath = path.join(this.projectRoot, "public", "wallet", "logo.png");
+      const logoBuffer = await fs.readFile(logoPath);
+
+      // Composite layers
+      const heroImage = await sharp(blackCanvas)
+        .resize(this.heroWidth, this.heroHeight)
+        .composite([
+          {
+            input: logoBuffer,
+            gravity: "center",
+            blend: "over",
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)